docs/UsersGuide/source/BuildingRunningTesting/Quickstart.rst

```diff
@@ -73,27 +74,56 @@ For a detailed explanation of how to build and run the SRW App on any supported
          cd ush
          cp config.community.yaml config.yaml
       
-      Users will need to open the ``config.yaml`` file and adjust the experiment parameters in it to suit the needs of their experiment (e.g., date, grid, physics suite). At a minimum, users need to modify the ``MACHINE`` parameter. In most cases, users will need to specify the ``ACCOUNT`` parameter and the location of the experiment data (see :numref:`Section %s <Data>` for Level 1 system default locations). Additional changes may be required based on the system and experiment. More detailed guidance is available in :numref:`Section %s <UserSpecificConfig>`. Parameters and valid values are listed in :numref:`Chapter %s <ConfigWorkflow>`. 
+      Users will need to open the ``config.yaml`` file and adjust the experiment parameters in it to suit the needs of their experiment (e.g., date, grid, physics suite). At a minimum, users need to modify the ``MACHINE`` parameter. In most cases, users will need to specify the ``ACCOUNT`` parameter and the location of the experiment data (see :numref:`Section %s <Data>` for Level 1 system default locations). 
+      
+      For example, a user on Gaea might adjust or add the following fields to run the 12-hr "out-of-the-box" case on Gaea using prestaged system data and :term:`cron` to automate the workflow: 
+
+      .. code-block:: console
+
+         user:
+           MACHINE: gaea
+           ACCOUNT: hfv3gfs
+         workflow:
+           EXPT_SUBDIR: run_basic_srw
+           USE_CRON_TO_RELAUNCH: true
+           CRON_RELAUNCH_INTVL_MNTS: 3
+         task_get_extrn_ics:
+           USE_USER_STAGED_EXTRN_FILES: true
+           EXTRN_MDL_SOURCE_BASEDIR_ICS: /lustre/f2/dev/role.epic/contrib  UFS_SRW_data/v2p2/input_model_data/FV3GFS/grib2/${yyyymmddhh}
```

Comment:
There is a random space in this path. Replace it with a '/' will return a full path to the grib2 files on Gaea.

Reply:
Fixed!